package.json

```diff
@@ -24,7 +24,7 @@
     "form-data": "3.0.0",
     "husky": "4.3.6",
     "merge": "2.1.0",
-    "semantic-release": "17.3.1",
+    "semantic-release": "17.3.0",
```

Comment:
Is er een reden voor de downgrade?

Reply:
Beyond being the same as some other registries - nope.

Reply:
Zal na deze merge wel door dependabot gebumped willen worden naar boven :D Zet het maar direct al op .1, spaart mij een PR merge achteraf